### ISSUE TITLE
Add option to change the animation duration for tabcontoller

### ISSUE THREAD

**Issue by guenth39** [Fintasys/emoji_picker_flutter#22]
I would love to change the animation duration from one page to another in the Config. We could just add an attribute `Duration animationDuration` or `Duration pageChangeDuration` to the config and pass it to the `animateTo` function [here](https://github.com/Fintasys/emoji_picker_flutter/blob/master/lib/src/default_emoji_picker_view.dart#L95).

**Comment by Fintasys** [comment]
@guenth39 Thank you for your feedback. Could consider that small change in next release. Can I ask you in what way you want to change it? More faster / slower?

**Comment by guenth39** [comment]
Hey Stefan,
thanks a lot for the reply. I would like it definitely faster :)

**Comment by Fintasys** [comment]
@guenth39 I have added the change to master and gonna make a new release soon. Would be nice if you give it a try when you have time
```
  emoji_picker_flutter:
    git:
      url: https://github.com/Fintasys/emoji_picker_flutter.git
      ref: master
```

**Comment by Fintasys** [comment]
Released with `v1.0.7`
